bigquery_storage/tests/system/test_reader_dataframe.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright 2019 Google LLC
+# Copyright 2018 Google LLC
```

Comment:
Should reverting the year back to 2018 be omitted?

If a file is renamed, should it be treated as a new file, or should the original Copyright year be preserved?

Reply:
I reverted this one because it shows up as a file move in the git history. I wasn't sure either, whether it would be considered a new file or not.

Reply:
Let's merge it as is then, and we can fix the year separately if needed.